llpc/tool/llpcCompilationUtils.cpp

```diff
@@ -396,6 +374,7 @@ Error processInputPipeline(ICompiler *compiler, CompileInfo &compileInfo, const
   if (EnableOuts() && !InitSpvGen())
     LLPC_OUTS("Failed to load SPVGEN -- cannot disassemble and validate SPIR-V\n");
 
+  compileInfo.pipelineType = pipelineState->pipelineType;
```

Comment:
nit: can we move this above `if (ignoreColorAttachmentFormats) {` so that all compile options are set in the same place?